src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx

```diff
@@ -243,7 +243,9 @@ Read more about each of our specific notification integrations.
 
     ## Prerequisites [#slack-prereqs]
 
-    - Your Slack workspace needs to have the [New Relic application](https://newrelic.slack.com/apps/AP92KQJS3-new-relic?tab=more_info) installed. The application must be approved by a workspace administrator before it can be individually installed by users.
+    - Your Slack workspace needs to have the [New Relic application](https://newrelic.slack.com/apps/AP92KQJS3-new-relic?tab=more_info) installed. The application must be approved by a workspace administrator before it can be individually installed by users. Please contact support@newrelic.com for App support
```

Comment:
```suggestion
    - Your Slack workspace needs to have the [New Relic application](https://newrelic.slack.com/apps/AP92KQJS3-new-relic?tab=more_info) installed. The application must be approved by a workspace administrator before it can be individually installed by users. Please contact support@newrelic.com for app support
```